.github/merge_request_template/Default.md

```diff
@@ -0,0 +1,26 @@
+## Describe your changes
+
+### JIRA link
+<!--- Please include a link to the JIRA ticket if there is one associated with -->
+
+### Design document
```

Comment:
I think we can leave this out, we have it in the API for context of folks outside of the squad, but I feel everyone in the squad should be familiar with the features and their design